rust_src/src/lib.rs

```diff
@@ -64,6 +64,7 @@ mod editfns;
 mod util;
 mod minibuf;
 mod cmds;
+mod fns;
```

Comment:
I'd put it in `objects.rs` instead.

Reply:
I wanted to keep the file structure the same as with original emacs sources, but I have no problem moving it to objects.rs . will do it right away. 

Thanks!

Reply:
We've sort of picked apart fns.c already since it is such a grab-bag. Thanks for the update!